create-api-diff.sh

```diff
@@ -26,8 +26,7 @@ if [ ! -d eclipse ];
 then
    echo "Downloading Eclipse"
    # Get basic Eclipse SDK
-   wget -q 'http://download.eclipse.org/eclipse/downloads/drops4/R-4.12-201906051800/eclipse-SDK-4.12-linux-gtk-x86_64.tar.gz'
-   tar -zxf eclipse-SDK-4.12-linux-gtk-x86_64.tar.gz
+   curl -m 600 -sL 'http://download.eclipse.org/eclipse/downloads/drops4/R-4.12-201906051800/eclipse-SDK-4.12-linux-gtk-x86_64.tar.gz' --output 'eclipse-SDK-4.12-linux-gtk-x86_64.tar.gz' && tar -zxf eclipse-SDK-4.12-linux-gtk-x86_64.tar.gz
```

Comment:
I suppose there are no stable simlinks available?

Reply:
none i know of